app/templates/rules.html

```diff
@@ -84,7 +84,12 @@ <h3 class="panel-title">
     <div class="left" style="float: left">
 
       <h5 title="Mapping">
-        Mapping: <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+        Mapping: <span ng-show="rule.mapping">
+            <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+            <!--<a href="/releases/#{{ rule.mapping }}"><b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b></a>-->
+            <a href="" ng-show="rule.mapping" ng-click="openReleaseDataModal(rule.mapping)">(view)</a>
+            <a href="/releases/#{{ rule.mapping }}">(edit)</a>
```

Comment:
just so I understand: does this mean we open the release blob in edit mode? So the 'view inline' is a view only modal and the actual link against the channel name is writeable link?